lasagne/layers/recurrent.py

```diff
@@ -230,18 +282,19 @@ def __init__(self, incoming, input_to_hidden, hidden_to_hidden,
         # Get the input dimensionality and number of units based on the
         # expected output of the input-to-hidden layer
         self.num_inputs = np.prod(input_shape[2:])
-        self.num_units = input_to_hidden.output_shape[-1]
+        self.hid_shape = input_to_hidden.output_shape
```

Comment:
Good call.  

> Or maybe set it to input_to_hidden.output_shape[1:]

I like the syntax of `hidden_to_hidden.output_shape` more because it's hard to know what to name "the hidden shape after the first dimension".